ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/handler/TestAbortedTxnCleaner.java

```diff
@@ -315,7 +327,6 @@ public void testAbortedCleaningWithThreeTxnsWithDiffWriteIds() throws Exception
     addDeltaFile(t, null, writeId1, writeId1, 2);
     addDeltaFile(t, null, writeId2, writeId2, 2);
 
-    ms.abortTxns(Collections.singletonList(openTxnId2));
```

Comment:
Why are we not aborting txns? This is what is tested here.

Reply:
I think it was a leftover from the first version of the change when I didn't check the compaction queue. Thank you for spotting it, re-added the missing line.